.cirrus.yml

```diff
@@ -1,7 +1,7 @@
 check_android_task:
   name: Run Android tests
   env:
-    API_LEVEL: 28
+    API_LEVEL: 29
```

Comment:
Not sure about the API_LEVEL put in .cirrus.yml; I think it's best to put it at most recent version (i.e API_LEVEL 31)